examples/Basic/index.android.js

```diff
@@ -11,20 +11,80 @@ import {
   StyleSheet,
   Text,
   View,
+  Dimensions,
+  Switch,
+  InteractionManager,
 } from 'react-native';
+import AndroidSegmented from 'react-native-segmented-android';
 
 import { BlurView } from 'react-native-blur';
 
+const BLUR_TYPES = ['xlight', 'light', 'dark'];
+
 class Basic extends Component {
   constructor() {
     super();
     this.state = {
-      viewRef: 0,
+      showBlur: true,
+      viewRef: null,
+      activeSegment: 2,
+      blurType: 'dark',
     };
   }
 
   imageLoaded() {
-    this.setState({ viewRef: findNodeHandle(this.refs.backgroundImage) });
+    // Workaround for a tricky race condition on initial load.
+    InteractionManager.runAfterInteractions(() => {
+      setTimeout(() => {
+        this.setState({ viewRef: findNodeHandle(this.refs.backgroundImage) });
+      }, 500);
+    });
+  }
+
+  _onChange(selected) {
+    this.setState({
+      activeSegment: selected,
+      blurType: BLUR_TYPES[selected],
+    });
+  }
+
+  renderBlurView() {
+    const tintColor = ['#ffffff', '#000000'];
+    if (this.state.blurType === 'xlight') tintColor.reverse();
+
+    return (
+      <View style={styles.container}>
+        <BlurView
+          viewRef={this.state.viewRef}
+          style={styles.blurView}
+
+          blurAmount={9}
+          blurType={this.state.blurType}
+
+          // blurRadius={20}
```

Comment:
I was thinking that this is example code that someone might be reading, so it could be useful to show the additional props on Android. I've added a header:

```
          // The following props are also available on Android:

          // blurRadius={20}
          // downsampleFactor={10}
          // overlayColor={'rgba(0, 0, 255, .6)'}   // set a blue overlay
```

Happy to remove all of those if you think that's better